include/eos/morphablemodel/PcaModel.hpp

```diff
@@ -193,6 +193,14 @@ class PcaModel
         return draw_sample(coeffs_float);
     };
 
+    template<std::size_t N>
+    Eigen::VectorXf draw_sample(std::array<double, N> coefficients) const
```

Comment:
I am not sure the use case is large enough for this to be in the `PcaModel` file. I would move it for now to the place where it's used (i.e. app or Ceres fitting).

Reply:
I can move it from here if you want, but there are many such functions in this file. As far as I understand the compiler will remove it from final code if the function will not be used in the program. Do you think it is really needed?